### PR TITLE
Bug/bs 334 large payload

### PR DESCRIPTION
**Pull request by AbstractFruitFactory**
https://radixdlt.atlassian.net/browse/BS-334

### CODE REVIEW

File: src/modules/atommodel/serializer/RadixSerializer.ts [hunk 1]
```diff
@@ -165,7 +165,7 @@ export class RadixSerializer {
     }
 
     public static toDSON(data: any): Buffer {
-        const enc = new cbor.Encoder()
+        const enc = new cbor.Encoder({ highWaterMark: 90000 })
```
Comment (MuncleUscles) [inline src/modules/atommodel/serializer/RadixSerializer.ts]: Maybe we should get in a comment here explaining what the `highWaterMark` thing is

File: test/integration/MetaDataAtom.spec.ts [hunk 2]
```diff
@@ -124,32 +124,34 @@ describe('RLAU-572: MetaData in Atoms', () => {
...
-            nodeConnection.submitAtom(atom).subscribe({
-                error: (e) => { done() },
-                complete: () => { done('Should have failed') },
+        return new Promise((resolve, reject) => {
+            buildTestAtom({
+                massive: hugeMetaData,
+            }).then((atom) => {
+                nodeConnection.submitAtom(atom).subscribe({
+                    error: (e) => { reject(e.message) },
+                    next: () => { resolve() },
```
Comment (MuncleUscles) [inline test/integration/MetaDataAtom.spec.ts]: Definitely don't want to resolve on `next` - it's going to fire even before it hits the node with the `PENDING` status, `complete` would work.

However this should really be rejected by the node, could you get a bug report into the RNET board? https://radixdlt.atlassian.net/secure/RapidBoard.jspa?rapidView=76&projectKey=RNET&view=planning

File: test/integration/MetaDataParticleGroup.spec.ts [hunk 3]
```diff
@@ -124,32 +124,34 @@ describe('RLAU-572: MetaData in ParticleGroups', () => {
...
-            nodeConnection.submitAtom(atom).subscribe({
-                error: (e) => { done() },
-                complete: () => { done('Should have failed') },
+        return new Promise((resolve, reject) => {
+            buildTestAtom({
+                massive: hugeMetaData,
+            }).then((atom) => {
+                nodeConnection.submitAtom(atom).subscribe({
+                    error: (e) => { reject(e) },
+                    next: () => { resolve() },
```
Comment (MuncleUscles) [inline test/integration/MetaDataParticleGroup.spec.ts]: Same issue as above

**Review by MuncleUscles — CHANGES_REQUESTED** [review]
The tests pass too early

File: test/integration/MetaDataAtom.spec.ts (hunk 2 above)
Comment (AbstractFruitFactory) [reply test/integration/MetaDataAtom.spec.ts]: Hmm, looked like the other tests used `next`

Comment (AbstractFruitFactory) [reply test/integration/MetaDataAtom.spec.ts]: I tried `complete` first but it didn't work

File: test/integration/Serializer.spec.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+describe('Token transfers', () => {
+    const identityManager = new RadixIdentityManager()
+    let identity1: RadixIdentity
+
+    before(async () => {
+        RadixLogger.setLevel('error')
+
+        const universeConfig = RadixUniverse.LOCALHOST
+        radixUniverse.bootstrap(universeConfig)
```
Comment (MuncleUscles) [inline test/integration/Serializer.spec.ts:24]: Should use trusted node bootstrapping

File: test/integration/Serializer.spec.ts [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    RadixIdentityManager,
+    RadixTransactionBuilder,
+    RadixLogger,
+    logger,
+    RadixIdentity,
+} from '../../src'
+
+const ERROR_MESSAGE = 'Local node needs to be running to run these tests'
+
+describe('Token transfers', () => {
```
Comment (MuncleUscles) [inline test/integration/Serializer.spec.ts]: The name here is wrong. This field is used for the `--grep` parameter

**Review by MuncleUscles — COMMENTED** [review]
Just tiny things left

**Pull request merged**